packages/core/src/routes/logto-config.ts

```diff
@@ -39,6 +39,16 @@ const getLogtoJwtTokenKey = (key: LogtoJwtTokenKeyType): LogtoJwtTokenKey =>
     ? LogtoJwtTokenKey.AccessToken
     : LogtoJwtTokenKey.ClientCredentials;
 
+const guardRequestBodyByTokenType = (tokenType: LogtoJwtTokenKeyType, body: unknown) => {
```

Comment:
please refer to the previous PR.  Can return a guarded type value instead of using as.